artemis/reporting/base/report.py

```diff
@@ -104,8 +104,9 @@ def alternative_with_ip_address(self) -> Optional["Report"]:
             report = copy.deepcopy(self)
             target_parsed = urllib.parse.urlparse(self.target)
             target_parsed_dict = target_parsed._asdict()
-            assert target_parsed.port is not None
-            target_parsed_dict["netloc"] = self.target_ip + ":" + str(target_parsed.port)
+            target_parsed_dict["netloc"] = self.target_ip + (
+                (":" + str(target_parsed.port)) if target_parsed.port else ""
+            )
```

Comment:
Prime example why I hate black. This code is unreadable, because of formatter. Please create helper variable here as a workaround.